networking/bgp.md

```diff
@@ -54,13 +54,24 @@ For a deeper look at common on-premises deployment models, see [Calico over IP F
 
 ### How to
 
-- [Disable the default BGP node-to-node mesh](#disable-the-default-bgp-node-to-node-mesh)
-- [Configure a global BGP peer](#configure-a-global-bgp-peer)
-- [Configure a per-node BGP peer](#configure-a-per-node-bgp-peer)
-- [Configure a node to act as a route reflector](#configure-a-node-to-act-as-a-route-reflector)
-- [View BGP peering status for a node](#view-bgp-peering-status-for-a-node)
-- [Change the default global AS number](#change-the-default-global-as-number)
-- [Change AS number for a particular node](#change-as-number-for-a-particular-node)
+- [Big picture](#big-picture)
```

Comment:
It looks like another cset / stash sneaked in here.

Reply:
At the moment this plugin only scans code blocks which are created when using one or three \`.

Reply:
I did write another one to fully address #3769 that replace or converts these characters globally.

Reply:
Why did this section change though? There's nothing to do with another character set in here, menu items are completely different.

Reply:
That was a mistake on my end instead of applying the plugin to pre_render I did it on post_render.